.gitignore

```diff
@@ -78,3 +78,4 @@ stryker.log
 # yalc
 .yalc/
 yalc.lock
+startnode.sh
```

Comment:
@ronaldmansveld - I'm assuming this is a local script you have... do you think it's worth adding a new section, like "# Shell scripts", and exclude all `.sh` files?

Reply:
It is, similar to yours. Not sure if we want to exclude all .sh files. Perhaps better to raise this in tech forum and discuss before making a decision.